lib/rpc/wallet.rb

```diff
@@ -97,7 +97,7 @@ def self.get_transfers(args={})
       h = Hash.new
       json = RPC::Client.request("get_transfers", options)
       json.map{|k, v|
-        h[k] = v.collect{|transfer| (RPC.config.transfer_clazz || "RPC::IncomingTransfer").constantize.new(transfer)}
+        h[k] = v.collect{|transfer| Module.const_get((RPC.config.transfer_clazz || "RPC::IncomingTransfer")).new(transfer) }
```

Comment:
did you test that LoC? i don't have any ruby right now on hand to test it.

Reply:
Yes, sure, it works.

Reply:
merged and pushed to rubygems already.